packages/esm-patient-chart-app/src/clinical-views/components/encounter-tile/encounter-tile.component.tsx

```diff
@@ -0,0 +1,100 @@
+import { CodeSnippetSkeleton, Tile, Column, Layer } from '@carbon/react';
+import React, { useMemo } from 'react';
+import styles from './tile.scss';
+import { groupColumnsByEncounterType } from '../../utils/helpers';
+import { useLastEncounter } from '../../hooks/useLastEncounter';
+import { LazyCell } from '../../../lazy-cell/lazy-cell.component';
+import { useTranslation } from 'react-i18next';
+import { type FormattedCardColumn } from '../../utils/encounter-tile-config-builder';
+import { useLayoutType } from '@openmrs/esm-framework';
```

Comment:
Please review our Coding Conventions on code organization and how imports should be ordered.